src/client/features/paint/pathway-node-metadata.js

```diff
@@ -1,81 +1,120 @@
 const React = require('react');
 const h = require('react-hyperscript');
+const _ = require('lodash');
+
+const { generateIdentifiersUrl } = require('../../common/external-service-data');
+const { ServerAPI } = require('../../services');
+
+const DEFAULT_NUM_NAMES = 3;
+const SUPPORTED_DB_LINKS = {
+  'Reactome': 'reactome',
+  'UniProt': 'uniprot knowledgebase',
+  'NCBI Gene': 'ncbi gene',
+  'HGNC Symbol': 'hgnc symbol'
```

Comment:
'hgnc symbol' is the id currently used to generate identifiers urls but it should actually be 'hgnc.symbol'